Brooklyn/Contexts/Preferences/PreferencesWindowController.swift

```diff
@@ -24,6 +24,7 @@ final class PreferencesWindowController: NSWindowController, NSWindowDelegate {
     @IBOutlet private weak var previewLabel: NSTextField!
     @IBOutlet private weak var randomOrderCheckBox: NSButton!
     @IBOutlet private weak var numberOfLoopsPopUp: NSPopUpButton!
+    @IBOutlet private weak var currentVersionLabel: NSTextField!
```

Comment:
Instead of a TextField, let's make this a button like the GitHub  and Author buttons. This one should redirect to `https://github.com/pedrommcarrasco/Brooklyn/releases`

Also, to make it coherent with the other buttons, please add the following icon 
[version-icon.zip](https://github.com/pedrommcarrasco/Brooklyn/files/3694444/version-icon.zip)

Reply:
Also forgot to mention, but the placement of the label looks 👌

Reply:
Than you for your input 🙌 

I changed the Label to a Button, added the image and added the action.

One problem though. The images provided are square (80x80) whereas the GitHub and Twitter images are 812x512, which gives a bit of padding between image and text.

I tried resizing the images from 80x80 to 512x512 and then added some additional padding so they ended up being 812x512 but my design skills are weak 😄 It still seems of as you can see on the attached screenshot....plus its probably not the right way to go, I mean, the images will just end up being blurred.

<img width="912" alt="Screenshot 2019-10-06 at 17 07 39" src="https://user-images.githubusercontent.com/117636/66271264-4859f100-e85c-11e9-9b89-ab45e1ac6ae6.png">

I can continue tweaking it, but if you have the original images, then maybe you could provide me with a new set of images matching the GitHub/twitter ones.

Hope I'm making sense 😄

Reply:
Yeah, I totally forgot how weird `AppKit` can be... In order to give that spacing without going with a custom `NSButton` I gave some paddings to the images 😅 

Here's a download link to all assets updated (please replace twitter and GitHub with these)
https://www.sendspace.com/file/po22ig

Reply:
Copy that, I'll replace the assets for all three images/buttons

> Yeah, I totally forgot how weird AppKit can be

Agreed, it looks and behaves similar to UIKit...until it doesn't and you're left wondering what just happened 😄